src/modules/shared/components/Tabs/ScrollableTabs/ScrollableTabs.module.scss

```diff
@@ -37,7 +42,7 @@ $scrollable-tabs-gradient-width-lg: $spacer-md + $scrollable-tabs-gradient-width
 
 	// Above :global
 	&--bordered {
-		border: 1px solid $zinc;
+		border: 1px solid $silver;
```

Comment:
`#ADADAD66` over `#fff` converts to `#dedede` as a solid value. Close to `$silver` but not quite. I'd opt to just transparentize `$zinc` by `0.6` instead. (https://rgbacolorpicker.com/hex-to-rgba)
<img width="990" alt="Screenshot 2022-04-13 at 10 33 03" src="https://user-images.githubusercontent.com/90180417/163134984-b9aa7757-4bbf-43e4-a78c-1eb50ab593f1.png">

Reply:
Dit is bewust omdat een semi-transparante border (`#ADADAD66`) toegepast op de zoekbalk én scrollable tabs ernaast voor twee verschillende grijstinten zorgt.